vsintegration/src/FSharp.Editor/Common/LanguageService.fs

```diff
@@ -48,7 +48,7 @@ type internal FSharpCheckerProvider
     ) =
     let checker = 
         lazy
-            let checker = FSharpChecker.Create()
+            let checker = FSharpChecker.Create(projectCacheSize = 200, keepAllBackgroundResolutions = false)
```

Comment:
@dsyme In VFPT, we provided an option in settings dialogs to override project cache size (default 50) https://github.com/fsprojects/VisualFSharpPowerTools/blob/03c613acd3aa67cb5e3a339d3dcf1c7c783f30c2/src/FSharpVSPowerTools/UI/GlobalOptionsPage.cs#L29-L33